### DISCUSSION TITLE
Pregabalin + CNS Depressants (alcohol, benzos, opioids)

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#92] [Drug Interactions]
Pregabalin increases the risk of overdose when taken with other CNS depressants.

Durand, L., O’Kane, A., Tierney, J., Cronly, M., Bennett, K. E., Kavanagh, Y., Keenan, E., & Cousins, G. (n.d.). Gabapentinoids in Ireland 2010 to 2020: An observational study of trends in gabapentinoid prescribing, law enforcement drug seizures and postmortem toxicology. British Journal of Clinical Pharmacology, n/a(n/a). https://doi.org/10.1111/bcp.15984

Evoy, K. E., Sadrameli, S., Contreras, J., Covvey, J. R., Peckham, A. M., & Morrison, M. D. (2021). Abuse and Misuse of Pregabalin and Gabapentin: A Systematic Review Update. Drugs, 81(1), 125–156. https://doi.org/10.1007/s40265-020-01432-7

Gomes, T., Greaves, S., van den Brink, W., Antoniou, T., Mamdani, M. M., Paterson, J. M., Martins, D., & Juurlink, D. N. (2018). Pregabalin and the Risk for Opioid-Related Death: A Nested Case-Control Study. Annals of Internal Medicine, 169(10), 732–734. https://doi.org/10.7326/M18-1136

Kriikku, P., & Ojanperä, I. (2021). Pregabalin and gabapentin in non-opioid poisoning deaths. Forensic Science International, 324, 110830. https://doi.org/10.1016/j.forsciint.2021.110830

**Comment by utaninja** [comment]
We should try to split up discussion pages for different combinations just for the future.
Breaking this down-
Durand, L., O’Kane, A., Tierney, J., Cronly, M., Bennett, K. E., Kavanagh, Y., Keenan, E., & Cousins, G. (n.d.). Gabapentinoids in Ireland 2010 to 2020: An observational study of trends in gabapentinoid prescribing, law enforcement drug seizures and postmortem toxicology. British Journal of Clinical Pharmacology, n/a(n/a). https://doi.org/10.1111/bcp.15984
> Seems that this study could go under benzos and opioids

Gabapentin: A Systematic Review Update. Drugs, 81(1), 125–156. https://doi.org/10.1007/s40265-020-01432-7
> Interesting study and good to note, but doesn't seem to me that it applies directly to any interaction.

Gomes, T., Greaves, S., van den Brink, W., Antoniou, T., Mamdani, M. M., Paterson, J. M., Martins, D., & Juurlink, D. N. (2018). Pregabalin and the Risk for Opioid-Related Death: A Nested Case-Control Study. Annals of Internal Medicine, 169(10), 732–734. https://doi.org/10.7326/M18-1136
> Seems to be for opioids

Kriikku, P., & Ojanperä, I. (2021). Pregabalin and gabapentin in non-opioid poisoning deaths. Forensic Science International, 324, 110830. https://doi.org/10.1016/j.forsciint.2021.110830
> Is this study demonstrating a increase in risk when taking a CNS depressant in addition to pregabalin?

This FDA warning demonstrates gabapentinoids cns depression concerns, especially under mixes. It cites 22 studies in total regarding this. This to me demonstrates more than enough that these should not be mixed. I'm unsure how we can best cite this, although perhaps we can simply cite this FDA warning?
https://www.fda.gov/drugs/drug-safety-and-availability/fda-warns-about-serious-breathing-problems-seizure-and-nerve-pain-medicines-gabapentin-neurontin

Pregabalin + Alcohol
Pregabalin and alcohol are CNS depressants and the combination of both makes it more likely to overdose. Both substances potentiate the ataxia and sedation caused by the other and can lead to unexpected loss of consciousness at higher doses. While unconscious, vomit aspiration is a risk if not placed in the recovery position. Memory blackouts are likely.

https://www.fda.gov/drugs/drug-safety-and-availability/fda-warns-about-serious-breathing-problems-seizure-and-nerve-pain-medicines-gabapentin-neurontin

Pregabalin + Benzos
Pregabalin and benzodiazipines are CNS depressants and the combination of both makes it more likely to overdose. Both substances potentiate the ataxia and sedation caused by the other and can lead to unexpected loss of consciousness at higher doses. While unconscious, vomit aspiration is a risk if not placed in the recovery position. Memory blackouts are likely.

Durand, L., O’Kane, A., Tierney, J., Cronly, M., Bennett, K. E., Kavanagh, Y., Keenan, E., & Cousins, G. (n.d.). Gabapentinoids in Ireland 2010 to 2020: An observational study of trends in gabapentinoid prescribing, law enforcement drug seizures and postmortem toxicology. British Journal of Clinical Pharmacology, n/a(n/a). https://doi.org/10.1111/bcp.15984
https://www.fda.gov/drugs/drug-safety-and-availability/fda-warns-about-serious-breathing-problems-seizure-and-nerve-pain-medicines-gabapentin-neurontin

Pregabalin + Opioids
Pregabalin and opioids are CNS depressants and the combination of both makes it more likely to overdose. Both substances potentiate the ataxia and sedation caused by the other and can lead to unexpected loss of consciousness at higher doses. While unconscious, vomit aspiration is a risk if not placed in the recovery position. Memory blackouts are likely.

Durand, L., O’Kane, A., Tierney, J., Cronly, M., Bennett, K. E., Kavanagh, Y., Keenan, E., & Cousins, G. (n.d.). Gabapentinoids in Ireland 2010 to 2020: An observational study of trends in gabapentinoid prescribing, law enforcement drug seizures and postmortem toxicology. British Journal of Clinical Pharmacology, n/a(n/a). https://doi.org/10.1111/bcp.15984
Gomes, T., Greaves, S., van den Brink, W., Antoniou, T., Mamdani, M. M., Paterson, J. M., Martins, D., & Juurlink, D. N. (2018). Pregabalin and the Risk for Opioid-Related Death: A Nested Case-Control Study. Annals of Internal Medicine, 169(10), 732–734. https://doi.org/10.7326/M18-1136
https://www.fda.gov/drugs/drug-safety-and-availability/fda-warns-about-serious-breathing-problems-seizure-and-nerve-pain-medicines-gabapentin-neurontin

**Reply by zelixir25** [reply]
yes i think the fda warning is certainly sufficient given the many sources they provide. the apa format citation for that would be:

FDA warns about serious breathing problems with seizure and nerve pain medicines gabapentin (Neurontin, Gralise, Horizant) and pregabalin (Lyrica, Lyrica CR). (2019). U.S. Food and Drug Administration. https://www.fda.gov/drugs/drug-safety-and-availability/fda-warns-about-serious-breathing-problems-seizure-and-nerve-pain-medicines-gabapentin-neurontin

**Comment by utaninja** [comment]
Pushed to https://github.com/TripSit/drugs/issues/321 (Pregabalin + Alcohol), https://github.com/TripSit/drugs/issues/322 (Pregabalin + Benzodiazepines), and I did a writeup for Pregabalin + Opioids - https://github.com/TripSit/drugs/issues/323)
